app/templates/sponsors.hbs

```diff
@@ -49,7 +61,19 @@
         <li>
           <figure>
             <a href={{sponsor.url}} rel="nofollow noopener noreferrer" target="_blank" class="well well-16/9">
-              <img src="/images/users/{{sponsor.image}}" alt="{{sponsor.name}} website" loading="lazy">
+              {{#if (eq (file-extension sponsor.image) "svg")}}
+                <img
+                  alt="{{sponsor.name}} website"
+                  src="/images/sponsors/{{sponsor.image}}"
+                  loading="lazy"
+                >
+              {{else}}
+                <ResponsiveImage
+                  @src="/images/sponsors/{{sponsor.image}}"
+                  @sizes="(min-width: 1008px) 234px, (min-width: 375px) 90px, 60px"
```

Comment:
```suggestion
                  @sizes="(min-width: 1008px) 234px, (min-width: 375px) 180px 90px, 60px"
```

Reply:
I don't think this is valid syntax, is it? See https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/sizes

And what exactly are you trying to achieve with this?

Tbh I haven't understood what exactly the problem is? I am aware of the images being too small, as I explained above. What else have you seen that needs fixing? Maybe post a screenshot?

Reply:
Yes sorry I don't actually know what the right syntax is 🙈  

Here is a set of images that look fine: 

Large: 
![deploy-preview-857--ember-website netlify app_ember-users_](https://user-images.githubusercontent.com/594890/129224480-f9e56508-961c-47e6-86d8-bc998b326067.png)

Smallest: 
![deploy-preview-857--ember-website netlify app_ember-users_ (1)](https://user-images.githubusercontent.com/594890/129224534-f1bd9cc4-a04b-4263-a5cd-aaffd31da104.png)

But here is the one that now looks different and the images are super small: 
![deploy-preview-857--ember-website netlify app_ember-users_ (3)](https://user-images.githubusercontent.com/594890/129224604-5e66a4a4-5ac1-4e97-8738-6168388b960a.png)